packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx

```diff
@@ -72,14 +97,40 @@ function SaveMnemonic({ className, onNextStep, onPreviousStep, seed }: Props): R
         </CopyToClipboard>
       </div>
       <VerticalSpace />
-      <ButtonArea>
+      <ButtonArea footer={footer}>
         <BackButton onClick={onPreviousStep} />
-        <Button onClick={onNextStep}>{t<string>('Next')}</Button>
+        <Button
+          isDisabled={!isSecretCopied}
+          onClick={onNextStep}
+        >
+          {t<string>('Next')}
+        </Button>
       </ButtonArea>
     </>
   );
 }
 
+const StyledCheckbox = styled(Checkbox)`
+  margin: 0;
+  font-size: 14px;
+  letter-spacing: 0.07em;
+  line-height: 145%;
+
+  && .checkbox-ui {
```

Comment:
Class referencing of nested components is quite nasty. Any chance we could fix this with changes in `<Checkbox>` itself?